### DISCUSSION TITLE
Announcement banner for breaking changes

### DISCUSSION THREAD

**Discussion by svrnm** [open-telemetry/opentelemetry.io#4300] [Tooling & Infrastructure]
Recently we got more often into the situation that a SIG introduced a breaking change or a change with major implications into their releases. So far we only have limited means to let people know (banner on the landing page, blog post). K8s has a feature I would like to introduce to the website as well, see an historic example here:

https://web.archive.org/web/20240104070231/https://kubernetes.io/docs/concepts/overview/

They have a large announcement banner that is active on all packages while this change is in flux. Of course we should use something like this sparsely, but it could help to put those changes right in front of people.

cc @mx-psi, @jpkrohling, @austinlparker 

@chalin, WDYT?

**Comment by mx-psi** [comment]
This change is under a feature gate on the Collector, which is still at alpha. Should we post the announcement after we promote to beta? (i.e. enabled by default)

**Comment by svrnm** [comment]
> This change is under a feature gate on the Collector, which is still at alpha. Should we post the announcement after we promote to beta? (i.e. enabled by default)

What I have posted above is a feature the website/docs do not have as of today, so we might not have it for the collector feature at all. But, independent of that, what you can do: create a blog post that outlines that change + an expected timeline, we put this in when it's in Beta and we add a banner to the website. IF we implement the feature discussed here we can see how we add this in additionally.

**Comment by mx-psi** [comment]
Okay, makes total sense! :) I think it is a good way of disseminating this knowledge and we should implement this feature

**Comment by chalin** [comment]
Sure, makes sense. How would you see "active on all packages" manifested on our website? Do you mean that all Collector-section pages would have a banner relevant for the Collector?

**Comment by svrnm** [comment]
I meant "active on all pages", it's a typo, but we might consider to have it only on collector pages if it is only affecting the collector, etc. I wonder how k8s website is implementing that.
